input/resources/au-core-client.xml

```diff
@@ -81,7 +81,7 @@
           <blockquote>
             <div>
               <ol>
-                <li>TBD.</li>
+               <li>For Authentication and Authorization, Systems <strong>SHOULD</strong> support any SMART App Launch and SMART Backend Service Authorization. When this is not avalable, Basic Authentication <strong>MAY</strong> be used. </li>
```

Comment:
This looks like proposed content not a clean up.

Reply:
done